Cargo.toml

```diff
@@ -303,9 +303,7 @@ ethexe-rpc = { path = "ethexe/rpc", default-features = false }
 ethexe-common = { path = "ethexe/common" }
 
 # Common executor between `sandbox-host` and `lazy-pages-fuzzer`
-sandbox-wasmi = { package = "wasmi", git = "https://github.com/gear-tech/wasmi", branch = "v0.13.2-sign-ext", features = [
-    "virtual_memory",
-] }
+wasmi = { package = "wasmi", version = "0.36"}
```

Comment:
I guess it can be bumped up to `0.38`. It also has a new [`extra-checks`](https://github.com/wasmi-labs/wasmi/pull/1217) feature. 

What do you think, @breathx?

Reply:
Should I enable this extra-checks feat?

Reply:
```
# Expected execution overhead is upt to 20%, if enabled.
#
# - Enable if your focus is on safety.
# - Disable if your focus is on execution speed.
extra-checks = []
```

Reply:
I think it would be great to have similar feature in our runner, but disable it by default: can’t find any pros at this point, especially with wasmi being such slow anyway

Reply:
Added disabled by default feat : `wasmi-extra-checks`